compiler/rustc_symbol_mangling/src/v0.rs

```diff
@@ -498,12 +499,9 @@ impl<'tcx> Printer<'tcx> for SymbolMangler<'tcx> {
             }
 
             ty::Pat(ty, pat) => {
-                // HACK: Represent as tuple until we have something better.
-                // HACK: constants are used in arrays, even if the types don't match.
-                self.push("T");
+                self.push("W");
```

Comment:
Can you update the [v0 mangling docs](https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/symbol-mangling/v0.md#mut-ptr-type) to incorporate the new manglings?